src/domain/occurrence/__tests__/CreateOccurrencePage.test.tsx

```diff
@@ -1232,6 +1233,148 @@ describe('enrolment type selector', () => {
   );
 });
 
+describe('auto acceptance for enrolments', () => {
+  it('renders auto acceptance message when auto accept is set on', async () => {
+    renderComponent({
+      mocks: [getEventMockedResponse({ autoAcceptance: false })],
+    });
+    // Wait for form to have been initialized
+    await screen.findByTestId('time-and-location-form');
+    // set as checked
+    act(() => userEvent.click(getFormElement('autoAcceptance')));
```

Comment:
I would expect that the `userEvent.click` helper is wrapped in an act and that the explicit `act` is not necessary here.

Possible changes that occur async should be flushed by `waitFor`. If this is not the case in practice, this approach is fine.

Reply:
I got warnings of it, but it may have been that I did not have the waitFor there next by then.

Reply:
yep, the act was needless. Removed it.